_sass/_outro.scss

```diff
@@ -0,0 +1,9 @@
+.outro {
+  height: 126px;
+  padding: 0;
+  background-size: 10px 126px;
+  background-repeat-x: repeat;
+}
+.outro {
+  background-image: url("/assets/img/jekyll-theme-reflection/prism_outro.svg");
```

Comment:
```suggestion
  background-image: url("/assets/img/svg/jekyll-theme-reflection/prism_outro.svg");
```

Reply:
🤔 shoot; tbh i am not familiar w/ proper Jekyll theming as independent structures:

> > please consider following the theming convention established by Jekyll proper
> 
> agreed, we would love to see the packaging of these up for others to use:
> 
> > Jekyll has an extensive theme system that allows you to leverage community-maintained templates and styles to customize your site’s presentation. Jekyll themes specify plugins and package up assets, layouts, includes, and stylesheets in a way that can be overridden by your site’s content.
> > —https://jekyllrb.com/docs/themes/

_Originally posted by @DerekNonGeneric in https://github.com/OpenINF/openinf.github.io/issues/1100#issuecomment-1913467899_

---

thinking we gonna have to move some things around soon; time for research now too

Reply:
> 🤔 shoot; tbh i am not familiar w/ proper Jekyll theming as independent structures

last night i did due diligence of reading up on what we are expected to provide as theme developers and my reaction was increasing "holy moly" &mdash; excited about the whole ordeal

i even made my very first RubyGems.org account &hellip; we're so back to Ruby community